api-metastore/src/main/java/org/zalando/nakadi/service/EventTypeService.java

```diff
@@ -541,6 +543,21 @@ private void validateAudience(final EventType original, final EventTypeBase even
         }
     }
 
+    private void validateEventAuthFieldUnchanged(final EventType original, final EventTypeBase eventTypeBase) throws
+            InvalidEventTypeException {
+        final EventAuthField originalEventAuthField = original.getEventAuthField();
+        final EventAuthField updatedEventAuthField = eventTypeBase.getEventAuthField();
+        if (updatedEventAuthField != null && originalEventAuthField != null) {
```

Comment:
What if `originalEventAuthField` is `null` and `updatedEventAuthField` is not?

Reply:
Or does it mean that admins can change this but normal users can only add it?

Reply:
> What if `originalEventAuthField` is `null` and `updatedEventAuthField` is not?

@rcillo I think that this case is perfectly valid - we need to set this field




> Or does it mean that admins can change this but normal users can only add it?

As I see right now - admins and normal users are in the same position. 
From the other hand I believe that nakadi admin should have extended capabilities, we can plan it for later

Reply:
In that case, users will be able to add `eventAuthField` to an event type which already doesn't have the field.
Only `nakadi` admins are allowed to change the field in case of an incident/requirement change.